joomla/joomla-cms
# [4.0] Use JLayout in view instead of the template files

### Summary of Changes

The goal of Joomla 4 is to move the template files (like default.php) into JLayouts everywhere, this PR is a first draft on that. It's keeping backward compatibility by loading the old template file / override if it exists.

The goal is to make layouts reusable, e.g. use them in views, modules etc. and to decouple the content from the view.

### Testing Instructions
Apply and make sure everything is still working with the b/c option.

Then

Remove / Rename the default.php in components/com_content/view/article/tmpl/default.php and see if the JLayout file is loaded and shows the article title.

### Documentation Changes Required

We need to update JDocs after finalising the approach.
